docs/hero-banner.md

```diff
@@ -29,11 +36,6 @@ Here are some cool activities to get you started!
 * name: How To Code a Video Game
 * imageUrl: /static/hero-gallery/how-to-code.png
 * youTubePlaylistId: PLMMBk9hE-SepfPK6z19t6aAfwbOQNY6K7
----
-* name: Join the Live Stream
```

Comment:
Was removing the stream part of the hero banner intentional?

Reply:
Yes. We were told that we can't link out to Twitch from our homepage anymore. We were supposed to take it off a while ago.

Reply:
Should we just change it to the YouTube link?